tests/StrategyHashDecoding.elm

```diff
@@ -101,6 +104,52 @@ validHashData =
                                 ]
                             )
             ]
+        , describe "V4 -> V5 migration"
+            [ test "custom portfolio using ranges - should give warning about portfolio simplification" <|
+                \() ->
+                    VersionedStrategy.loadStrategy "NDt7ImkiOltbMywxMF0sWzEzLDIwXSxbMTksMzBdLFsyMSwyMV0sWzE5LDE5XSxbMTEsMTFdLFs3LDddLFs1LDVdLFsyLDJdLFsxLDFdLFswLDEwXV0sImgiOnsiYSI6MywiYiI6WyIwIl0sImMiOlsxXSwiZCI6WzAsMjAwXSwiZSI6WzJdLCJnMSI6MX0sImoiOltbMCwyMDBdLFswLDIwMF0sWzAsMjAwXSxbMCwyMDBdLFswLDIwMF0sWzAsMjAwXSxbMCwyMDBdLFswLDIwMF0sWzAsMjAwXSxbMCwyMDBdLFswLDIwMF1dLCJrIjp7Im8iOjB9LCJsIjp7Im0iOjB9fQ=="
+                        |> withDecodedStrategy
+                            (Expect.all
+                                [ \( _, warnings ) ->
+                                    warnings
+                                        |> Expect.equal
+                                            [ "strategie měla nastavenu vámi definovanou strukturu portfolia, která musela být zjednodušena:"
```

Comment:
This is the kind of warning user will see if he used ranges when restoring strategy from url.